website/docs/tutorial/set-up-data-platform.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Set up your data platform
+id: set-up-data-platform
+description: "Set up your environment to get started using dbt."
+---
+
+Now that you have some context for this tutorial, let's get your environment set up. Check out the table of contents to the left to set up your environment on BigQuery, Snowflake, or Redshift. These guides are helpful for anyone with an existing data platform or starting from scratch.
```

Comment:
```suggestion
Now that you have some context for this tutorial, let's get your environment set up. Check out the table of contents to the left to set up your environment on BigQuery, Snowflake, or Databricks. These guides are helpful for anyone with an existing data platform or starting from scratch.
```
quick-fix: the three tutorials we have ready are BQ, SF, and Databricks. We will definitely loop in Redshift once we get that one ready.